cmd/agent/dist/conf.d/systemd.d/conf.yaml.example

```diff
@@ -8,18 +8,29 @@ instances:
   - unit_names:
       - <UNIT_NAME>
 
+    ## @param unit_regexes - list of strings - optional
```

Comment:
I can't find this being used anywhere. Where does this comes from ?

Reply:
you right, it's a miss.